motr/client_init.c

```diff
@@ -1678,7 +1683,8 @@ int m0_client_init(struct m0_client **m0c_p,
 	}
 
 	if (conf->mc_is_addb_init) {
-		char buf[64];
+		char buf[128];
+		const char *addb_stob_location;
```

Comment:
initialise with NULL, so that if it is m0ut or kernel module will be as good as previous implementation